src/languages/es.js

```diff
@@ -224,6 +224,8 @@ export default {
         editComment: 'Editar commentario',
         deleteComment: 'Eliminar comentario',
         deleteConfirmation: '¿Estás seguro de que quieres eliminar este comentario?',
+        addEmojiReaction: 'Añadir una reacción emoji',
+        addReactionTooltip: 'Añadir una reacción...',
```

Comment:
Hm this looks different than the other one. I think this is just 3 `.` instead of the `...` symbol? Let's update this one to match